openedx/core/djangoapps/user_authn/views/login_form.py

```diff
@@ -251,6 +251,7 @@ def login_and_registration_form(request, initial_mode="login"):
             'is_account_recovery_feature_enabled': is_secondary_email_feature_enabled(),
             'is_multiple_user_enterprises_feature_enabled': is_multiple_user_enterprises_feature_enabled(),
             'enterprise_slug_login_url': get_enterprise_slug_login_url(),
+            'enterprise_enable': enterprise_enabled(),
```

Comment:
Since this is a boolean variable, it would be great if we prefix it with is `is_enterprise_enable`. Also in JS as well `isEnterpriseEnable`.